Source/DafnyCore/DooFile.cs

```diff
@@ -131,7 +130,7 @@ public class ManifestData {
     }
 
     var success = true;
-    var relevantOptions = currentCommand.Options.ToHashSet();
+    var relevantOptions = options.Options.OptionArguments.Keys.ToHashSet();
```

Comment:
Is this still correct, given the comment immediately below? i.e. is `OptionArguments.Keys` the set of valid parameters for the current command, or just the options explicitly provided on the CLI or in project files?

Reply:
This is still correct, yes